src/platforms/common/configuration/options.mdx

```diff
@@ -595,6 +595,14 @@ An optional property that configures which downstream services receive the `sent
 
 </ConfigKey>
 
+<ConfigKey name="trace-propagation-targets" supported={["node"]}>
+
+An optional property that controls which downstream services receive tracing data, in the form of a `sentry-trace` and `baggage` headers attached to any outgoing HTTP requests.
+It contains a list of URLs or regex against which URLs are matched.
+If this option is not provided, trace data is attached to every outgoing request from the instrumented client.
```

Comment:
I know that this an optional property but should we still add a sentence that by default/if it's not specified, trace data is sent everywhere?

Reply:
Are you suggesting that we remove that last sentence, or are you suggesting that we add a sentence that says it attaches trace data to every request? If latter, we already have that last sentence (line 602).

Reply:
Oh sorry, I must have missed that. Massive brainfart